internal/pkg/auth/storage_test.go

```diff
@@ -219,8 +363,21 @@ func TestSetGetAuthFieldKeyring(t *testing.T) {
 		t.Run(tt.description, func(t *testing.T) {
 			keyring.MockInit()
 
+			p := print.NewPrinter()
+			if tt.activeProfile != "" {
+				err := config.SetProfile(p, tt.activeProfile)
+				if err != nil {
+					t.Errorf("set profile: %v", err)
+				}
+			} else {
+				err := config.UnsetProfile(p)
+				if err != nil {
+					t.Errorf("unset profile: %v", err)
+				}
+			}
```

Comment:
Do we really need to set and unset profiles in tests? I think it's not needed for this function and for `TestSetGetAuthFieldEncodedTextFile`

Reply:
But in any case, these are actually creacting files in the user config dir and actually setting and unsetting profiles for the user, without cleaning up after themselves

Reply:
removed the set/unset from all test cases